<?xml version="1.0" ?>

<launch>

  <!-- Bringup joystick node -->
  <node pkg="joy" type="joy_node" name="joystick" respawn="false" >
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.05" />
  </node>

  <!-- Bringup nexus_teleop_node -->
  <node pkg="nexus_base_ros" type="nexus_teleop_joy" name="teleop_joy" respawn="false" output="screen" >
    <param name="axis_linear" value="1" type="int" />
    <param name="axis_angular" value="0" type="int" />
  </node>

  <!-- Bringup nexus_base -->
  <node pkg="rosserial_python" type="serial_node.py" name="nexus_base" respawn="false" output="screen" >
    <param name="port" value="/dev/ttyUSB0" />
    <param name="baud" value="57600" />
  </node>
  
  <?ignore
  <!-- Bringup nexus_base_controller node -->
  <node pkg="nexus_base_ros" type="nexus_base_controller" name="base_controller" respawn="false" />
  ?>  

  <!-- Bringup nexus_base_controller node -->
  <node pkg="nexus_base_ros" type="nexus_base_controller" name="base_controller" output="screen" respawn="false" />

</launch>



